x-pack/auditbeat/module/system/packages/_meta/data.json

```diff
@@ -0,0 +1,30 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "metricset": {
+        "module": "system",
+        "name": "packages",
+        "rtt": 115
+    },
+    "system": {
+        "packages": {
+            "package": [
+                {
+                    "arch": "amd64",
+                    "installtime": "0001-01-01T00:00:00Z",
```

Comment:
That's a long time ago

Reply:
Right, we can come up with a better example package, but for now the `packages` metricset is disabled, so I made no effort to keep the data up to date.